torch/onnx/symbolic_helper.py

```diff
@@ -661,17 +700,19 @@ def _select_helper(g, self, dim, index, apply_reshape=True):
     return g.op("Gather", self, index, axis_i=dim)
 
 
+@_beartype.beartype
 def _slice_helper(g, input, axes, starts, ends, steps=None, dynamic_slice=False):
     if GLOBALS.export_onnx_opset_version <= 9:
-        from torch.onnx.symbolic_opset9 import _slice as _slice9
-
-        return _slice9(g, input, axes, starts, ends)
+        # from torch.onnx.symbolic_opset9 import _slice as _slice9
+        _slice = opset9._slice
+        return _slice(g, input, axes, starts, ends)
```

Comment:
Could you call opset9.slice here and opset10.slice below directly in these cases?